### PR TITLE
Add additional timer for NGD new features

### PR DESCRIPTION
**Pull request by keziming**
The indentations show the level of subroutines. If two calls in the same level are separated between a blank line, it means the calls are sequential. Otherwise, only one call is chosen at a time in the code. The calls with parenthesis means it is not a V2 default call but added by new features.

Currently, only calls separate default and new feature are chosen in those two subroutines. We can add more later, but current setting is to figure out which parts cost most resources with simplicity.   

Timer at modal_aero_amicphys.F90

modal_aero_amicphys_intr

	call mam_amicphys_1gridcell

       call mam_amicphys_1subarea_cloudy
       call mam_amicphys_1subarea_clear

                     call mosaic_gasaerexch_1subarea_intr (mosaic)
                           call mosaic_box_aerchemistry
                           call mam_soaexch_vbs_1subarea 
                     call mam_gasaerexch_1subarea 
                     	   call mam_soaexch_vbs_1subarea (vbs SOA)
                           call mam_soaexch_1subarea

                     call mam_rename_1subarea_strat (MAM5)
                     call mam_rename_1subarea (MAM4)

                     call mam_newnuc_1subarea

                     call mam_coag_1subarea

                     call mam_pcarbon_aging_1subarea

modal_aero_wateruptake.F90

     call modal_aero_wateruptake_dr
              call modal_aero_wateruptake_strat_sub (MAM5)
              call modal_aero_wateruptake_sub

### CODE REVIEW

**Review by mingxuanwupnnl — APPROVED** [review]
I would suggest add a timer for mam_soaexch_vbs_1subarea within mosaic_gasaerexch_1subarea_intr

**Comment by keziming** [comment]
> I would suggest add a timer for mam_soaexch_vbs_1subarea within mosaic_gasaerexch_1subarea_intr

Just add it. Thanks!

**Review by tangq — APPROVED** [review]
The changes look great to me. Thanks for implementing this so quickly - Really helpful for the NGD tuning tasks.

I'd suggest double check the v3 smoke-test time breakdown table on [this page](https://acme-climate.atlassian.net/wiki/spaces/NGDAP/pages/3564961793/2022-11-11+Meeting+notes+-+v3+Integration) and be sure that major NGD costs are included in the timer. I am not familiar with the aerosol code. @hwangacme , @mingxuanwupnnl , and @mkshriva should be able to help if you are not sure about some parts.

**Review by hwangacme — APPROVED** [review]

File: components/eam/src/chemistry/modal_aero/modal_aero_amicphys.F90 [hunk 1]
```diff
@@ -970,7 +970,7 @@ subroutine modal_aero_amicphys_intr(                             &
          misc_vars_aa,                            &
          Hconc_sav, awater,         & !!MW: to save aerosol pH (dsj+zlu)
          troplev(i)          ) !kzm: troplev for stratospheric aerosol cal.
-
+      call t_startf ('mam_amicphys_1gridcell')
```
Comment (wlin7) [inline components/eam/src/chemistry/modal_aero/modal_aero_amicphys.F90:973]: This should be t_stopf('mam_amicphys_1gridcell')

File: components/eam/src/chemistry/modal_aero/modal_aero_amicphys.F90 [hunk 2]
```diff
@@ -1504,7 +1505,7 @@ subroutine mam_amicphys_1gridcell(          &
          misc_vars_aa_sub(jsub),                     & 
          Hconc_sav , awater,                          & !MW: to save aerosol pH (dsj+zlu)
          troplev_i) !kzm:for strat. aerosol cal.
-
+      call t_startf ('amicphys_1subarea_clear')
```
Comment (ambrad) [inline components/eam/src/chemistry/modal_aero/modal_aero_amicphys.F90:1508]: Another t_startf -> t_stopf change is needed here.

Comment (keziming) [reply components/eam/src/chemistry/modal_aero/modal_aero_amicphys.F90:1508]: thank you for pointing out that.

**Pull request merged**